### PR TITLE
Extended SSH Proxy Support

### PR DESCRIPTION
**Pull request by jluebbe**
**Description**
This PR implements support for using a SSH proxy host for connections to the coordinator and network resources. Note that so far only SerialDriver, NetworkPowerDriver (most backends), ModbusCoilDriver and OneWirePIODriver have been modified to use the ProxyManager.

This should also solve the use-case described in #186.

**Checklist**
- [ ] Documentation for the feature
- [ ] Tests for the feature 
- [ ] Add a section on how to use the feature to doc/usage.rst
- [x] CHANGES.rst has been updated
- [x] PR has been tested
- [x] Man pages have been regenerated

### CODE REVIEW

File: labgrid/driver/modbusdriver.py [hunk 1]
```diff
@@ -15,13 +16,18 @@ class ModbusCoilDriver(Driver, DigitalOutputProtocol):
...
         super().__attrs_post_init__()
         self._module = import_module('pyModbusTCP.client')
-        port = "502"
-        host = self.coil.host
-        if ':' in host:
-            host, port = host.split(':', 1)
+        self.client = None
+
+    def on_activate(self):
+        import onewire
```
Comment (Bastian-Krause) [inline labgrid/driver/modbusdriver.py]: Why is this needed in the ModbusCoilDriver?

File: labgrid/util/proxy.py [hunk 2]
```diff
@@ -1,21 +1,35 @@
+import os
+
+from urllib.parse import urlsplit, urlunsplit
+
 from .ssh import sshmanager
 
 from ..resource.common import NetworkResource, Resource
```
Comment (Bastian-Krause) [inline labgrid/util/proxy.py]: `NetworkResource` is not used anymore, remove the import.

File: labgrid/driver/modbusdriver.py (hunk 1 above)
Comment (jluebbe) [reply labgrid/driver/modbusdriver.py]: This was a copy/paste error.

File: labgrid/util/proxy.py (hunk 2 above)
Comment (jluebbe) [reply labgrid/util/proxy.py]: OK.
